R/non-overlap-measures.R

```diff
@@ -310,13 +316,15 @@ Tau_BC <- function(A_data, B_data, condition, outcome,
           condition = condition, outcome = outcome, 
           baseline_phase = baseline_phase,
           ES = "Tau_BC", improvement = improvement,
-          SE = SE, confidence = confidence)
+          SE = SE, confidence = confidence,
+          report_correction = report_correction)
 }
 
 calc_Tau_BC <- function(A_data, B_data, 
                         improvement = "increase", 
                         SE = "unbiased", CI = TRUE,
-                        confidence = .95, ...) {
+                        confidence = .95, report_correction = FALSE, 
```

Comment:
Is "report_correction" same as "pretest_trend" you mentioned in the comment?

Reply:
No. The report_correction argument controls whether additional information about the trend correction is included in the output. There should be a separate pretest_trend argument.